src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: $width-of-items;
```

Comment:
You're using a variable `$width-of-items` here, but it's not defined anywhere in the file. Make sure to import the file that contains this variable or define it in this file. This is a bug.